utils/docker/examples/client/daos-client_agent/el8/daos-bash.in

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# set -x
+set -e -o pipefail
+
+if [[ "$(id -u)" != "0" ]] ; then
+	echo "[ERROR] daos-bash can only be run as root"
+fi
+
+CWD="$(realpath "$(dirname $0)")"
```

Comment:
(lint) CWD appears unused. Verify use (or export if used externally). [SC2034]
(lint) Double quote to prevent globbing and word splitting. [SC2086]